src/sidebar.js

```diff
@@ -106,6 +106,7 @@ module.exports = {
             },
           ],
         },
+        "kratos/self-service/flows/user-logout",
```

Comment:
From notion, as requested by @aeneasr 
<img width="393" alt="image" src="https://user-images.githubusercontent.com/5540255/206454158-3a92b7a8-be77-49f9-b68a-643d9202bdfc.png">

Reply:
;(
not a fan of this

Reply:
Logout is now on the same level like login, registraiton, profile settings, account recovery.

The self-service section is completely dissolved. Since registration and logout are always self-service, there's no point really in having it

Reply:
I guess this is resolved then. I have also moved the verification & recovery docs to the same location. So it does look a bit better now.